ydb/docs/en/core/concepts/datamodel/_includes/table-name-rules.md

```diff
@@ -0,0 +1,32 @@
+## Table naming rules
+
+Every table in {{ ydb-short-name }} has a name. In YQL statements, table names are specified by identifiers that can be enclosed in backticks or not. For more information on identifiers, refer to [{#T}](../../../yql/reference/syntax/lexer.md#keywords-and-ids).
+
+Table names in {{ ydb-short-name }} must meet the following requirements:
+
+- Table name length must not exceed 255 characters.
+- Table names must not contain dots only.
+- Tables cannot be created in the system folder (`.sys`).
+- Table names can include the following characters:
```

Comment:
Let's move up the list of allowed characters to be the first bullet, as it is the most practical part, and the rest are more like corner cases.